src/cfnlint/data/ExtendedSpecs/all/01_spec_patch.json

```diff
@@ -119,5 +119,22 @@
     "op": "replace",
     "path": "/ResourceTypes/AWS::ResourceGroups::Group/Properties/Tags/ItemType",
     "value": "Tag"
+  },
+  {
+    "op": "add",
+    "path": "/PropertyTypes/AWS::SSM::Association.ParameterValues",
```

Comment:
@PatMyron this is what it was previously.  It used to be in the spec but got dropped for some reason in this release.

Reply:
copying the other snippet for reference:
```json
        "Parameters": {
          "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ssm-association.html#cfn-ssm-association-parameters",
          "UpdateType": "Mutable",
          "Required": false,
          "Type": "Map",
          "ItemType": "ParameterValues"
        },
```